contracts/land/LANDRegistry.sol

```diff
@@ -479,4 +472,87 @@ contract LANDRegistry is Storage, Ownable, FullAssetRegistry, ILANDRegistry {
     assembly { size := extcodesize(addr) }
     return size > 0;
   }
+
+  //
+  // LAND Ping
+  //
+
+  /**
+   * @dev Set date from when the LAND Ping feature should be enabled
+   * @param _gracePeriod Desired amount of time in miliseconds to enable feature
```

Comment:
Based on how you are using it on `hasDecayed`, it seems to be a specific date in seconds.

Solidity `block.timestamp` is in `seconds`.